### ISSUE TITLE
LtiLibrary.AspNet does not support cross platform ASP.NET Core

### ISSUE THREAD

**Issue by steelheaddigital** [LtiLibrary/LtiLibrary#43]
The LtiLibrary.AspNet package still has a dependency on System.Web which does not exist in ASP.NET Core.

**Comment by andyfmiller** [comment]
I'm working on converting LtiLibrary.AspNet to .NET Core in the vs17 branch. So far both LtiLibrary.Core and LtiLibrary.AspNet are converted to target net451 and netstandard1.6 (the same two that Microsoft.AspNetCore.Mvc targets). Most changes are in the API controllers to support Outcomes and ToolConsumerProfile. Outcomes.v1 is working. The others should be working soon.

As you might guess from the branch name, I've also switched to Visual Studio 2017 and the new project file format.

**Comment by andyfmiller** [comment]
I decided to create two new projects on the vs17 branch: LtiLibrary.NetCore and LtiLibrary.AspNetCore. I'll revert LtiLibrary.AspNet back to the way it was before I started this work. And I may remove the NetCore stuff from LtiLibrary.Core.

LtiLibrary.NetCore and LtiLibrary.AspNetCore are now converted. I tested all the basic functions: Launch, Outcomes 1, Outcomes 2, and ToolConsumerProfile. I haven't looked at Content-Item yet, but I don't expect any issues.

I will do some more testing and create NuGet packages before I close this issue.
